### DISCUSSION TITLE
Provide Android Java bindings

### DISCUSSION THREAD

**Discussion by ddobrev** [libvips/libvips#1994] [Ideas]
Android's native scaling algorithms are quite poor so a fast scaling library is going to be more than useful.

**Comment by jcupitt** [comment]
Good idea, though I won't be able to try myself, unfortunately.

**Comment by KishorJena** [comment]
If somebody knows about jni ndk and all he she should create a java android binding for libvips or libwebp. There is no simple library for basic manipulation of animated webp and gifs. If someone will create this then it will spread to other frameworks and langugaes like Flutter, React native etc.. I read instructions on libwebp. It's command line works like charm and it has pre compiled stuff out there. I will definitely do in future as I don't know even core of ndk jni c++ things. 
I loved the libvips in Sharp. It is super and best for some use cases. I saw people need to use ImageMagick, ffmpeg which has more then enough functionalities and perform slower for basic operations like gif and webp resizing and  conversion.
